src/http/client.rs

```diff
@@ -399,7 +408,7 @@ impl EspHttpConnection {
                     })?;
                     esp!(unsafe { esp_http_client_set_redirection(self.raw_client) })?;
                     esp!(unsafe {
-                        esp_http_client_open(self.raw_client, self.request_content_len as _)
+                        esp_http_client_open(self.raw_client, self.request_content_len as i32)
```

Comment:
Please just change the type of `EspHttpConnection::request_content_len` to `i64`, as you originally started doing. Sorry for asking you to do other stuff - I did not realize, that `request_content_len` is a private field which is not visible to the user.

Reply:
Okay, change type to `i64`.
[a7d084c](https://github.com/esp-rs/esp-idf-svc/pull/310/commits/a7d084cb4c792362c1dbeb33b91476cd1ac8492c)